docs/articles/new-expensify/payments/Request-Money.md

```diff
@@ -7,31 +7,127 @@ redirect_from: articles/request-money/Request-and-Split-Bills/
 
+1. Click on the ‘+’ **Create** button
+2. Click **Request Money**
+3. Click **Manual** (to create an expense manually)
+ - Enter the amount on the receipt > click **Next**
+ - Type the workspace or recipient email
+ - Enter a:
+   - Description
+   - Merchant (required)
+   - Date (required)
+   - Category
+   - Tag
+   - Enable billable, if applicable
+4. Click **Scan** (to SmartScan an expense)
+  - Drag and drop the receipt into Expensify to start the SmartScan process
+  - The SmartScan process will auto-populate the merchant, date, and amount
+  - Enter a:
+    - Description
+    - Merchant (required)
+    - Date (required)
+    - Category
+    - Tag
+    - Enable billable, if applicable
+5. Click **Distance** (to create a distance expense)
+- Click Start and enter the starting location of your trip
+- Click Stop and enter the ending location of your trip
+- Choose a workspace to submit the expense to (usually your company workspace if one exists)
+- Enter a:
+  - Description
+  - Merchant (required)
+  - Date (required)
+  - Category
+  
+# How to Request Money from a friend
+
+1. Select the Green + button and choose **Request Money**
+2. Take a photo of your receipt if you have one, or manually enter the amount and select **Next**
+3. Search for the user or enter their email
+4. Enter a reason for the request (optional)
+- If you manually entered the amount, you can also input the merchant and date of the original payment
+6. Click **Request**
+7. If you change your mind, all you have to do is select **Cancel**
+8. Your friend can settle up outside of Expensify or pay you via Venmo or PayPal.me
+
+  
 # How to Split a Bill
-- Select the Green **+** button and choose **Split Bill**
-- Enter the total amount for the bill and click **Next**
-- Search for users or enter their emails and **Select**
-- Enter a reason for the split
-- The split is then shared equally between the attendees
+
+1. Select the Green **+** button and choose **Request Money**
+2. Take a photo of your receipt if you have one, or manually enter the total amount for the bill and select **Next**
+3. Search for users or enter their emails and select **Split**
+- Select all users you want to split the bill with
+4. Select **Add to split**
+5. Enter a reason for the split (optional)
+- If you manually entered the amount, you can also input the merchant and date of the original payment</li>
+6. Select **Split** $[amount]
+7. This will send a payment request to each person
+
+# How to submit an expense to your employer
+
+1. Sign in to New Expensify on the web at www.new.expenisfy.com or via the Android or iOS mobile app
+2. Look for a green dot on the left-hand side of the app to locate the new expense
+
+<img width="369" alt="image" src="https://github.com/Expensify/App/assets/51066321/fdb3252f-bb02-4335-9346-163f0878673c">
+
+4. Review the **Next Step** indication at the top of the expense report
+5. If the **Next Step** is to submit the report, click the **Submit** button
```

Comment:
```suggestion
3. Review the **Next Step** indication at the top of the expense report
4. If the **Next Step** is to submit the report, click the **Submit** button
```

It doesn't look good right now

![image](https://github.com/Expensify/App/assets/21549771/f83b1622-6f85-4cbd-8a55-a953f9781c05)


cc @rushatgabhane

Reply:
@rushatgabhane it's been 2 weeks

Reply:
@lakchote sorry im not the author, so i don't have access to the branch

Reply:
i agree with your suggestion

Reply:
@Christinadobrzyn you have conflicts to handle and this suggestion to take care of. If that's easier for you, you might want to close this PR and open a new one starting from latest changes in this file. Thanks!